### PR TITLE
Use Chrome 59 for PNG cheatsheet rendering

### PR DESCRIPTION
**Pull request by bootstraponline**
- Zero dependencies to render a PNG now.
- PDF rendering should work also however Chrome 59 doesn't capture the text. Seems like a bug.

### CODE REVIEW

File: Scripts/cheatsheet/render.rb [hunk 1]
```diff
@@ -14,34 +14,22 @@
...
 
-  # remove right hand scrollbars by cropping.
-  crop_width = screenshot.width - 20
-  screenshot.crop!(0, 0, crop_width, screenshot.height)
-  screenshot.save(cheatsheet_png, :best_compression)
-ensure
-  web_driver.quit
-end
+# run_command "#{chrome} --headless --disable-gpu --print-to-pdf=#{cheatsheet_pdf} file://#{cheatsheet_html}"
+run_command "#{chrome} --headless --hide-scrollbars --disable-gpu --screenshot=#{cheatsheet_png} --window-size=1024,2550 file://#{cheatsheet_html}"
```
Comment (wuhao5) [inline Scripts/cheatsheet/render.rb]: does this require version 59? can you also add a check and assert that too?

Comment (bootstraponline) [inline Scripts/cheatsheet/render.rb]: > does this require version 59?

yes

> can you also add a check and assert that too?

sure

File: Scripts/cheatsheet/render.rb [hunk 2]
```diff
@@ -14,34 +14,21 @@
...
-  screenshot = eyes_browser.fullpage_screenshot
-
-  # remove right hand scrollbars by cropping.
-  crop_width = screenshot.width - 20
-  screenshot.crop!(0, 0, crop_width, screenshot.height)
-  screenshot.save(cheatsheet_png, :best_compression)
-ensure
-  web_driver.quit
-end
+# run_command "#{chrome} --headless --disable-gpu --print-to-pdf=#{cheatsheet_pdf} file://#{cheatsheet_html}"
```
Comment (wuhao5) [inline Scripts/cheatsheet/render.rb]: remove this line?

File: Scripts/cheatsheet/render.rb (hunk 1 above)
Comment (wuhao5) [inline Scripts/cheatsheet/render.rb]: cool! I patched and tested it locally, worked for me!

**Comment by bootstraponline** [comment]
> will it make more sense to rewrite this in bash?

if we use bash then i'd prefer a one liner and dropping all the conditional checking. fine with that?

**Comment by wuhao5** [comment]
>  a one liner and dropping all the conditional checking
Yes, I like one liner! I think you could still do some checks just so when anyone runs it, it can give meaningful errors.

also @sid-github or @tirodkar , WDYT?

**Comment by bootstraponline** [comment]
Ok, added bash version.

**Pull request merged**